### ISSUE TITLE
npm audit report

### ISSUE THREAD

**Issue by szmarczak** [jaredwray/keyv#69]
Running `npm audit` gave me this:

```
  Moderate        SQL Injection
  Package         sql
  Patched in      No patch available
  Dependency of   @keyv/mysql [dev]
  Path            @keyv/mysql > @keyv/sql > sql
  More info       https://nodesecurity.io/advisories/662

  Moderate        SQL Injection
  Package         sql
  Patched in      No patch available
  Dependency of   @keyv/postgres [dev]
  Path            @keyv/postgres > @keyv/sql > sql
  More info       https://nodesecurity.io/advisories/662

  Moderate        SQL Injection
  Package         sql
  Patched in      No patch available
  Dependency of   @keyv/sqlite [dev]
  Path            @keyv/sqlite > @keyv/sql > sql
  More info       https://nodesecurity.io/advisories/662

  Low             Prototype Pollution
  Package         lodash
  Patched in      >=4.17.5
  Dependency of   @keyv/mysql [dev]
  Path            @keyv/mysql > @keyv/sql > sql > lodash
  More info       https://nodesecurity.io/advisories/577

  Low             Prototype Pollution
  Package         lodash
  Patched in      >=4.17.5
  Dependency of   @keyv/postgres [dev]
  Path            @keyv/postgres > @keyv/sql > sql > lodash
  More info       https://nodesecurity.io/advisories/577

  Low             Prototype Pollution
  Package         lodash
  Patched in      >=4.17.5
  Dependency of   @keyv/sqlite [dev]
  Path            @keyv/sqlite > @keyv/sql > sql > lodash
  More info       https://nodesecurity.io/advisories/577
```

Should we be worried about this?

**Comment by lukechilds** [comment]
Oh and 

>Also, what about dropping Node 4 & 6?

Yes definitely!

Are you by any chance interested in taking any of these tasks on?

**Comment by lukechilds** [comment]
Thanks, you're an absolute hero 🙌

**Comment by adityapatadia** [comment]
Let me know what are your thoughts on dropping that support? Do we just do major version release and write in release notes that the old version is compatible with those versions but new ones are not?

**Comment by lukechilds** [comment]
>Do we just do major version release and write in release notes that the old version is compatible with those versions but new ones are not?

Yeah, that should be fine.

**Comment by sustained** [comment]
Is this still planned (migration to e.g. Knex), or?

**Comment by szmarczak** [comment]
Wish I just had more time, currently I'm working on Got and it's quite hard for me to work on many projects that require much attention... Feel free to pick this up.

**Comment by adityapatadia** [comment]
Isn't GOT stable released already? Just want to know what is being added to it.

**Comment by szmarczak** [comment]
@adityapatadia https://github.com/sindresorhus/got/issues/

**Comment by szmarczak** [comment]
Stable release does not mean that developing will stop. There's still plenty of work to do.

**Comment by HunteRoi** [comment]
Same here!
```powershell
> npm audit

                       === npm audit security report ===                        


                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           


  Low             Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      >=4.17.5                                                      

  Dependency of   @keyv/sqlite                                                  

  Path            @keyv/sqlite > @keyv/sql > sql > lodash                       

  More info       https://npmjs.com/advisories/577                              


  High            Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      >=4.17.11                                                     

  Dependency of   @keyv/sqlite                                                  

  Path            @keyv/sqlite > @keyv/sql > sql > lodash                       

  More info       https://npmjs.com/advisories/782                              


  High            Prototype Pollution                                           

  Package         lodash                                                        

  Patched in      >=4.17.12                                                     

  Dependency of   @keyv/sqlite                                                  

  Path            @keyv/sqlite > @keyv/sql > sql > lodash                       

  More info       https://npmjs.com/advisories/1065                             


  Moderate        SQL Injection                                                 

  Package         sql                                                           

  Patched in      No patch available                                            

  Dependency of   @keyv/sqlite                                                  

  Path            @keyv/sqlite > @keyv/sql > sql                                

  More info       https://npmjs.com/advisories/662                              

found 4 vulnerabilities (1 low, 1 moderate, 2 high) in 529 scanned packages
  4 vulnerabilities require manual review. See the full report for details.
```

**Comment by jaredwray** [comment]
@szmarczak - we are now supporting 12, 14, and 16 and validated.
